### PR TITLE
[Bugfix] zoom to feature (from the attribute table) goes to far away

### PR DESCRIPTION
**Pull request by rldhont**
Fixes https://github.com/3liz/lizmap-web-client/issues/3474

If the layers are not in EPSG:4326 to zoom button in the attribute table goes the map to far away.

And `getDataAndFillAttributeTable` and `getFeatureData` as iso

Funded by 3liz https://3liz.com/

### CODE REVIEW

File: assets/src/legacy/attributeTable.js [hunk 1]
```diff
@@ -335,8 +343,14 @@ var lizAttributeTable = function() {
...
                     }
-                    lizMap.config.layers[layerName]['columns'] = responses[1].columns;
-                    buildLayerAttributeDatatable(layerName, tableSelector, responses[0].features, responses[1].aliases, responses[1].types, allColumnsKeyValues, callBack);
+                    layerConfig['featureCrs'] = 'EPSG:4326';
+                    if (namedRequests?.['describeFeatureType']) {
+                        const describeFeatureTypeResponse = responses[namedRequests['describeFeatureType']];
+                        layerConfig['aliases'] = describeFeatureTypeResponse.aliases;
+                        layerConfig['types'] = describeFeatureTypeResponse.types;
+                        layerConfig['columns'] = describeFeatureTypeResponse.columns;
+                    }
```
Comment (mdouchin) [inline assets/src/legacy/attributeTable.js:352]: Shouldn't we test that `describeFeatureTypeResponse` contains `aliases`, `types` and `columns`

**Review by mdouchin — CHANGES_REQUESTED** [review]

File: assets/src/legacy/attributeTable.js (hunk 1 above)
Comment (rldhont) [reply assets/src/legacy/attributeTable.js:352]: This request with json output format has been created for that - it is a dedicated lizmap request.

Comment (rldhont) [reply assets/src/legacy/attributeTable.js:352]: I have added a test, to check each properties from the request.

**Review by mdouchin — APPROVED** [review]

**Comment by rldhont** [comment]
I have added tests for https://github.com/3liz/lizmap-web-client/pull/3495

**Pull request merged**